flutter/example/fastlane/Fastfile

```diff
@@ -0,0 +1,94 @@
+default_platform(:ios)
+
+# Download dSYMs does not work with app_store_connect_api_key yet. Therefore
+# We use username and password to authenticate. This feature is blocked by the
+# App Store Connect API team, see https://github.com/fastlane/fastlane/discussions/17485#discussioncomment-125619
+# When this is fixed we can remove SPACESHIP_SKIP_2FA_UPGRADE, FASTLANE_USER and FASTLANE_PASSWORD
+ENV["SPACESHIP_SKIP_2FA_UPGRADE"] = "1"
```

Comment:
This is not going to work anymore sadly. We need to set `FASTLANE_SESSION`. You can generate one with [spaceauth](https://docs.fastlane.tools/best-practices/continuous-integration/#spaceauth). Checkout our internal notion page for Cocoa for more details.

Reply:
This is a good reminder that this setup should be in a state where we can easily maintain across repos. gittree is one option. other thoughts?

It's a lot of different bits and pieces to copy paste across 5 or more repos

Reply:
I agree. As already discussed it would be nice to get it up and running here and then try to share the script. We could use `import_from_git` from Fastlane https://docs.fastlane.tools/actions/import_from_git/